adage/analyze/tests.py

```diff
@@ -366,8 +366,24 @@ def setUp(self):
             **self.test_experiment))
 
         # Create a few annotation types to retrieve with the API
-        self.at_count = 10
-        factory.create(AnnotationType, self.at_count)
+        ann_type = AnnotationType(
+            typename="type name % one",
```

Comment:
Keep in mind that the Django test is not launching a web server. I think this means that URLs are not *actually* being generated and used, so this is not exactly a live API test, but `api_client.get()` is the right test harness for this.

These are examples I found that broke the test in the same place that the fixtureless-generated data were doing. More importantly, the fix I applied to the tests made these examples stop breaking the test.

Note the comment in the [docs](http://django-tastypie.readthedocs.io/en/latest/testing.html#get) that the method "Optionally accepts a `data` kwarg, which in the case of `GET`, lets you send along `GET` parameters. " This was the hint I needed that we should be doing it that way rather than building the URL manually.